### PR TITLE
add directory_check

### PR DESCRIPTION
**Pull request by ndushay**
It can be useful to know if a mounted shared file system has been set up when an application relies on it.

### CODE REVIEW

File: lib/ok_computer/built_in_checks/directory_check.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    attr_accessor :directory, :writable
+
+    # Public: Initialize a new directory check.
+    #
+    # directory - the path of the directory.  Can be relative or absolute.
+    # writable - true if directory should allow writes;  false if not.
+    def initialize(directory, writable = true)
+      raise ArgumentError if directory.blank?
+      raise ArgumentError if writable.class != TrueClass && writable.class != FalseClass
```
Comment (pbyrne) [inline lib/ok_computer/built_in_checks/directory_check.rb]: This feels like it's more protection than is warranted. In theory, any truthy/falsey value here would do what you want. In practice, I'm not terribly concerned about someone misconfiguring their own check by passing invalid values.

Comment (ndushay) [inline lib/ok_computer/built_in_checks/directory_check.rb]: ok, took that line (and the corresponding test) out.

**Comment by ndushay** [comment]
might this be ready for a merge?

**Comment by pbyrne** [comment]
v1.13.0, which includes this change, is now available. Thanks for the contribution!

**Pull request merged**